pkg/sslpolicy.go

```diff
@@ -13,10 +13,10 @@ import (
 // necessary to create our globally enforced SSL Policy.
 func NewSslPolicy(name string) compute.SslPolicy {
```

Comment:
This function is to create a new SSL Policy, what if we only want to update the existing SSL policy?

Reply:
Then we should have a new policy or make change via terraform, I think using go code to manage the policy while we can do it much easier with terraform is not very efficient if we don't use it very often.

Reply:
This part will check if the SSL policy exists. Otherwise, it will create the new one.
https://github.com/commercetools/gcp-ssl-policy-asserter/blob/1b436c7dc0e3e1add35ef487d7f91b74bdd2761b/pkg/sslpolicy.go#L72

Reply:
It only asserts the SSL policy is existed, not if the Policy is changed.

Reply:
@tuananhnguyen-ct we don't need to create SSL policy in TF anymore, it needs to be cleaned up in ops-terraform.